external-crates/move/crates/move-compiler/tests/custom_rules/constant_naming.move

```diff
@@ -0,0 +1,20 @@
+module 0x42::M {
+    // Correctly named constants
+    const MAX_LIMIT: u64 = 1000;
+    const MIN_THRESHOLD: u64 = 10;
+    const MIN_U64: u64 = 10;
+
+    // // Incorrectly named constants
```

Comment:
This says incorrectly named, but they shouldn't trigger a warning? I find that a bit confusing

Reply:
refactored